docs/modules/ROOT/pages/scripts.adoc

```diff
@@ -13,12 +51,10 @@ Useful if you need to deploy an account in devnet, for previous declaration:
 async def run(nre):
   accounts = await nre.get_accounts(predeployed=True)
   declarer_account = accounts[0]
-  print(await declarer_account.declare("Account"))
+  print(await declarer_account.declare("Account", nile_account=True))
```

Comment:
i'd add a small note explaining what `nile_account` is (it tells nile to use its pre-packed artifacts)

Reply:
Added a comment on the script.